python/python-hello-world/README.md

```diff
@@ -7,129 +7,48 @@
 [Service](https://kubernetes.io/docs/concepts/services-networking/service/). The Deployment contains a
 [Flask-based](http://flask.pocoo.org/) web server that simply prints "Hello World".
 
-![NOTE](./img/noun_Excitement_267_3BB300.png) | We will be using [Kubernetes terminology](https://kubernetes.io/docs/reference/glossary/?fundamental=true) throughout in this document
------|------
-
 ----
 
-## Table of Contents
-1. [Getting Started](#getting-started)
-2. [Included Files](#included_files)
-3. [Using Cloud Code](#using-cloudcode)
-    1. [Set up a Google Kubernetes Engine Cluster](#set-up-a-google-kubernetes-engine-cluster)
-    2. [Launch a Deployment](#launch-a-deployment)
-    3. [View Container Logs](#view-container-logs)
-    4. [Debug Your Code](#debug-your-code)
-    5. [Open a Terminal in Your Container](#open-a-terminal-in-your-container)
-    6. [Tasks](#tasks)
+# Table of Contents
+
+## Cloud Code for Visual Studio Code
+1. [Getting Started](#getting-started])
+2. [What's in the box](https://cloud.google.com/code/docs/vscode/quickstart#whats_in_the_box)
+3. Using Cloud Code
+    * [Set up a Google Kubernetes Engine Cluster](https://cloud.google.com/code/docs/vscode/quickstart#creating_a_google_kubernetes_engine_cluster)
+    * [Deploy the app](https://cloud.google.com/code/docs/vscode/quickstart#deploying_your_app)
+    * [Continuous Deployment](https://cloud.google.com/code/docs/vscode/quickstart#initiating_continuous_deployment)
+    * [View Container Logs](#view-container-logs)
+    * [Debug Your Code](#debug-your-code)
```

Comment:
I think you need cloud.google.com links for these items too.

Reply:
Good catch. Thanks!